recon_test_pack/scatter_simulation_new.par

```diff
@@ -0,0 +1,21 @@
+Scatter Simulation := 
```

Comment:
name of file?

Reply:
I am sorry are you referring to the ```_new```

Reply:
yes. seems that we can just use the existing scatter_simulation.par?